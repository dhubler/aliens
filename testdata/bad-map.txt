Boston south=NewYork
NewYork north=NewHaven