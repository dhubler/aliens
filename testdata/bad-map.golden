Boston north=NewYork south=NewYork
NewYork north=Boston south=Boston
